corehq/apps/export/management/commands/process_skipped_pages.py

```diff
@@ -161,7 +161,7 @@ def _get_unprocessed_pages(self, export_archive_path, extract_to_path):
             else:
                 raise CommandError('Unexpected page filename: {}'.format(page_filename))
 
-            doc_count = int(sh.wc('-l', page_path).split(' ')[0])
+            doc_count = int(sh.wc('-l', page_path).strip().split(' ')[0])
```

Comment:
This didn't even seem to be accurately retrieving the version because there was lots of leading white space that needed to be trimmed down before splitting the output.